src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -68,6 +76,13 @@ static ucs_config_field_t uct_cuda_copy_md_config_table[] = {
      ucs_offsetof(uct_cuda_copy_md_config_t, pref_loc),
      UCS_CONFIG_TYPE_ENUM(uct_cuda_pref_loc)},
 
+    {"PREF_ALLOCATOR", "fabric",
```

Comment:
IMO we can rename it to "ALLOC", and support a priority list, similar to https://github.com/openucx/ucx/blob/4e10990960ca4a92d94e713ca8c155016ae51cee/src/uct/base/uct_iface.c#L823

Reply:
@yosefe Is the suggestion to give highest priority to `fabric-pinned` memory allocations followed by `legacy-pinned`? Maybe as follows:
```C
{"ALLOC", "fabric-pinned,legacy-pinned", 
```
I would leave out fabric-host (EGM) in the default list and would make the allocation method only if user explicitly requests it.